docs-src/python/how-to-skip-time-set-up-in-python.md

```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-skip-time-set-up-in-python
+title: How to set up time skipping in Python
+sidebar_label: Set up time skipping
+description: Skip time from the test server automatically.
+tags:
+  - python
+  - how-to
+  - time-skipping
+---
+
+To set up a time skipping environment, import [`WorkflowEnvironment`](https://python.temporal.io/temporalio.testing.WorkflowEnvironment.html) from the [`testing`](https://python.temporal.io/temporalio.testing.html) module.
```

Comment:
This doesn't setup an environment, this just does an import. That doesn't really do anything.